src/main/java/org/embulk/parser/jsonpath/JsonpathParserPlugin.java

```diff
@@ -177,15 +219,15 @@ public void run(TaskSource taskSource, Schema schema,
 
     private Map<Column, String> createJsonPathMap(PluginTask task, Schema schema)
     {
-        ImmutableMap.Builder<Column, String> builder = ImmutableMap.builder();
+        Map<Column, String> columnMap = new HashMap<>();
         for (int i = 0; i < schema.size(); i++) {
             ColumnConfig config = getSchemaConfig(task).getColumn(i);
-            JsonpathColumnOption option = config.getOption().loadConfig(JsonpathColumnOption.class);
+            JsonpathColumnOption option = CONFIG_MAPPER_FACTORY.createConfigMapper().map(config.getOption(),JsonpathColumnOption.class);
```

Comment:
If `CONFIG_MAPPER_FACTORY.createConfigMapper()` is called several times, you can just store it, jfyi.
```
private final ConfigMapper CONFIG_MAPPER = CONFIG_MAPPER_FACTORY.createConfigMapper();
```

Reply:
Sharing ConfigMapper instance. Additionally, I added `static`. Because It used in `private static TimestampFormatter[] newTimestampColumnFormatters` method.
I copied this method from the [csv parser](https://github.com/embulk/embulk/blob/d551c29727ead3b08bb017c1bfe8fdfdeb237d44/embulk-parser-csv/src/main/java/org/embulk/parser/csv/CsvParserPlugin.java#L487-L506)

```
private static final ConfigMapper CONFIG_MAPPER = CONFIG_MAPPER_FACTORY.createConfigMapper();
```